### PR TITLE
feat: shareable new content - LESQ-915

### PR DESCRIPTION
**Pull request by JBR90**
## Description

Music year: 1959

- Activate share button on lesson overview page for new lessons, 

## How to test

1. Go to https://deploy-preview-2544--oak-web-application.netlify.thenational.academy
All post legacy lessons (new lessons) should have share all button on lesson overview page.

## Screenshots

How it used to look (delete if n/a):
image

How it should now look:
image

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx [hunk 1]
```diff
@@ -209,7 +209,7 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
         track={track}
         analyticsUseCase={analyticsUseCase}
         isNew={isNew}
-        isShareable={isLegacyLicense && !expired}
+        isShareable={!expired}
```
Comment (njblore) [inline src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx:212]: think we should remove the code that adds 'coming soon' as well

Comment (JBR90) [reply src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx:212]: Yeah good point

**Review by njblore — APPROVED** [review]

**Review by rightsaidjames — APPROVED** [review]
Nice and straightforward, works as expected

**Pull request merged**